app/views/shared/_announcements.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="announcement container-fluid <%= Settings.announcement&.html_class || "yoo doggie" %>">
```

Comment:
Is the OR operator needed here?  It can just be nil if html_class is not defined.

Reply:
Ha ha, absolutely not necessary. Glad you caught this.

Yes, my plan was to inject the yml from config like in catalog. Planning to try this out in the QA environment.